src/workflows-app/RunDetails.js

```diff
@@ -56,10 +58,11 @@ export const BaseRunDetails = (
   const { captureEvent } = useMetricsEvent();
 
   const [sasToken, setSasToken] = useState('');
-  const showLogModal = useCallback((modalTitle, logsArray) => {
+  const showLogModal = useCallback((modalTitle, logsArray, tesLog) => {
     setShowLog(true);
     setLogsModalTitle(modalTitle);
     setLogsArray(logsArray);
+    setTemplateTesLog(tesLog);
```

Comment:
As discussed in standup - can we make this flexible to it can take either a "template" log path or a TES log directory? That way we can switch Cromwell to storing the log directory metadata and this will continue working.